devconf-outreachy/source/index.rst

```diff
@@ -0,0 +1,126 @@
+
+.. Becoming OpenStack Developer via Outreachy slides file, created by
+   hieroglyph-quickstart on Wed Jan  2 14:19:51 2019.
+
+
+Becoming OpenStack Developer via Outreachy
+==========================================
+
+Presented by: Aija Jauntēva, Dmitry Tantsur, Richard Pioso, Ilya Etingof
+
```

Comment:
maybe add here another slide or include in the title slide who is who (intern, mentor, etc.) to introduce us at the very beginning?

Reply:
I am thinking, we could just do some introductions in the beginning...? Or do you think this'd justify a slide?

/me being modest and keeps blushing while typing in this comment.